exercises/vehicles/tests/sql_testkit.sh

```diff
@@ -20,7 +20,10 @@ CURRENT_SCENARIO=""
 CURRENT_TEST=""
 INDENT_LEVEL=0
 
+DEBUG_LOGGING=false
```

Comment:
Might want to consider setting this as an argument to pass, but I don't feel strongly about it.